hack/update-codegen.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
```

Comment:
Do you really wish to keep -x?  This is usually just for debugging.

Reply:
no, forgotten thanks